docs/pages/management/admin/self-signed-certs.mdx

```diff
@@ -0,0 +1,216 @@
+  which you can download by visiting the
+  [customer portal](https://dashboard.gravitational.com/web/login).
+
+  ```code
+  $ tctl version
+  # Teleport v(=teleport.version=) go(=teleport.golang=)
+  
+  $ tsh version
+  # Teleport v(=teleport.version=) go(=teleport.golang=)
+  ```
+
+</TabItem>
+</Tabs>
+
+## How to use self-signed certs with Teleport binaries
+
+### `teleport`
+
+When running a Teleport service with `teleport`, if the service you are starting is configured to point to
+the Teleport Proxy Service endpoint and the Proxy Service is using self-signed certificates, then `teleport` will need
+to be run with the `--insecure` flag to disable verification of the
+Proxy Service TLS certificate. This is the case when:
+- The Teleport config file `auth_servers` setting is set to the Proxy Service endpoint:
+  - `auth_servers: ["tele.example.com:443"]` or
+  - `auth_servers: ["tele.example.com:3080"]`
+- Teleport is started with the `--auth-server` flag pointing to the Proxy Service endpoint:
+  - `teleport [app | db] start --auth-server=tele.example.com:443` or
+  - `teleport [app | db] start --auth-server=tele.example.com:3080`
+
+The way you make your Teleport service run with TLS certificate verification disabled depends on how you are
+running it: via the `teleport` CLI, using a Helm chart, or via systemd:
+<Tabs>
+
+    <TabItem label="Using the CLI">
+    When running `teleport` from the command line, pass the `--insecure` flag to disable
+    TLS certificate validation. For example:
+    ```sh
+    $ teleport start -c /etc/teleport.yaml --insecure
+    $ teleport app start -c /etc/teleport.yaml --insecure
+    $ teleport db start -c /etc/teleport.yaml --insecure
+    ```
+    Without the `--insecure` flag, you will see an error message that looks like
+    `x509: “tele.example.com” certificate is not trusted`.
+    </TabItem>
+
+    <TabItem label="Helm chart">
+    For deploying a cluster, set
+    [extraArgs](../../reference/helm-reference/teleport-cluster.mdx#extraargs)
+    to include the extra argument: `--insecure`:
+    <Tabs>
+      <TabItem label="values.yaml">
+      ```yaml
+      extraArgs:
+      - "--insecure"
+      ```
+      </TabItem>
+      <TabItem label="--set">
+      ```code
+      $ --set "extraArgs={--insecure}"
+      ```
+      </TabItem>
+    </Tabs>
+    
+    
+    For a Helm Kube Agent, set the 
+    [insecureSkipProxyTLSVerify
+    ](../../reference/helm-reference/teleport-kube-agent.mdx#insecureskipproxytlsverify)
+    flag to `true`:
+    <Tabs>
+      <TabItem label="values.yaml">
+      ```yaml
+      insecureSkipProxyTLSVerify: true
+      ```
+      </TabItem>
+      <TabItem label="--set">
+      ```code
+      $ --set insecureSkipProxyTLSVerify=true
+      ```
+      </TabItem>
+    </Tabs>
+    </TabItem>
+
+    <TabItem label="systemd">
+    Edit the `systemd` Teleport unit file to include `--insecure` in the `ExecStart` line, for example:
```

Comment:
Maybe add an instruction to run `systemctl status teleport` to get the path to the unit file?

Reply:
Good idea, added